Dockerfile

```
@@ -44,5 +44,6 @@ COPY supervisord_startup.sh /usr/local/sbin/
 COPY supervisord.conf /etc/
 COPY update-certs-rpms-if-present.sh /etc/cron.hourly/
 COPY cron.d/* /etc/cron.d/
+RUN chmod 644 /etc/cron.d/fetch-crl
```

```suggestion
RUN chmod go-w /etc/cron.d/* /etc/cron.hourly/* /etc/supervisord.conf /usr/local/sbin/*
```
Let's make sure none of the other files are world-writeable either.